packages/labs/task/src/task.ts

```diff
@@ -10,7 +10,7 @@ import {ReactiveControllerHost} from '@lit/reactive-element/reactive-controller.
 export type TaskFunction<D extends [...unknown[]], R = any> = (
   args: D
 ) => R | typeof initialState | Promise<R | typeof initialState>;
-export type DepsFunction<D extends [...unknown[]]> = () => D;
+export type ArgsFunction<D extends [...unknown[]]> = () => D;
```

Comment:
this is a breaking change btw, and maybe the only one. Do you want to export a `DepsFunction` alias so we don't have to release a new major?

Reply:
Done.